tests/tools/go.mod

```diff
@@ -1,16 +1,19 @@
 module github.com/containers/storage/tests/tools
 
-go 1.18
+go 1.23.0
 
 require (
 	github.com/cpuguy83/go-md2man v1.0.10
```

Comment:
BTW this version is also 6 years old, but I’ll leave that update to someone who wants to investigate differences in the generated `nroff` syntax.